pages/api/upload_album.py

```
@@ -1221,20 +1263,139 @@ def get_folder_info_by_name():
+        
+    full_path = os.path.join(LOCAL_STORAGE_PATH, clean_path)
+    return send_file(full_path)
+
+# Lấy đường dẫn tuyệt đối của project
+BASE_DIR = Path(__file__).resolve().parent.parent.parent
+STORAGE_PATH = BASE_DIR / 'storage'
+PUBLIC_PATH = BASE_DIR / 'public'
+
+@app.route('/static/icon_folder/<path:filename>')
+def serve_icon(filename):
+    print(f"=== Debug serve_icon ===")
```

Ruff (0.8.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)